nosql-db/mongodb-reactive/src/test/resources/test.properties

```diff
@@ -1,3 +1,5 @@
 ts.app.log.enable=true
 ts.database.log.enable=true
 ts.database.openshift.use-internal-service-as-url=true
+# store MongoDB data in 'emptyDir' volume to work-around write permission issue
+ts.database.openshift.template=/mongo-openshift-deployment-template.yml
```

Comment:
wow, I didn't know we can do **that**!

Reply:
I was actually looking for a way I can just customize default template, but it is not possible ATM and I believe it will change when Michal Jurc finishes migration to Fabric8 Java client (from OC tool).

Reply:
So then we can refactor this.

Reply:
@mjurc fyi